catalog/spatial/registry/registry-policy-plugin/src/main/java/org/codice/ddf/registry/policy/RegistryPolicyPlugin.java

```diff
@@ -159,7 +159,7 @@ private void parsePermissionsFromString(List<String> permStrings,
 
     private PolicyResponse getWritePolicy(Metacard input, Map<String, Serializable> properties) {
         HashMap<String, Set<String>> operationPolicy = new HashMap<>();
-        if (Requests.isLocal(properties) && RegistryUtility.isRegistryMetacard(input)) {
+        if (Requests.isLocal(properties) && input != null && RegistryUtility.isRegistryMetacard(input)) {
```

Comment:
It might be better to update `RegistryUtility.isRegistryMetacard()` to return false for a null metacard.